source/renderer/app/themes/mixins/layers.scss

```diff
@@ -1 +1,3 @@
 $dialog-z-index: 200;
+$loader-backdrop-z-index: 5;
```

Comment:
5 is enough? 

<img width="660" alt="image" src="https://user-images.githubusercontent.com/2539345/157697675-893839b2-fa6a-4929-8874-e0ee0399ecf9.png">

Reply:
This is something else. But lets fix it too.

The news feed has a 200 z-index. With < 200, that tooltip goes underneath.

Reply:
And for your second image: after evaluating and finding that the third party tooltip we are using is set to 9999, I am setting the news feed to a level above that. Are we in agreement that the news feed should be the top most layer in every case?

Reply:
cc @daniloprates

Reply:
It's also possible to pass a zIndex property to the tooltip component to have a lower value.